TeacherWorkout.Api/appsettings.Development.json

```diff
@@ -8,6 +8,25 @@
   },
   "ConnectionStrings": {
     //connection string for local environment
-    "TeacherWorkoutContext": "host=localhost;Port=5432;Database=teacher_workout;User Id=docker;Password=docker;"
-  }
+    "TeacherWorkoutContext": "host=localhost;Port=5432;Database=teacher_workout;User Id=docker;Password=docker;",
+    "IdentityDbConnectionString": "host=localhost;Port=5432;Database=teacher_workout;User Id=docker;Password=docker;"
+  },
+  "JwtConfig": {
+    "Secret": "",
```

Comment:
We should add some values here or load from .env. I was getting errors when trying to start the project in development mode after pulling the branch locally.